src/vs/workbench/contrib/terminal/browser/xterm/quickFixAddon.ts

```diff
@@ -115,8 +119,12 @@ export class TerminalQuickFixAddon extends Disposable implements ITerminalAddon,
 		decoration?.onRender((e: HTMLElement) => {
 			if (!this._decorationMarkerIds.has(decoration.marker.id)) {
 				this._currentQuickFixElement = e;
-				e.classList.add(DecorationSelector.QuickFix, DecorationSelector.Codicon, DecorationSelector.CommandDecoration, DecorationSelector.XtermDecoration);
+				e.classList.add(DecorationSelector.QuickFix, DecorationSelector.LightBulb, DecorationSelector.Codicon, DecorationSelector.CommandDecoration, DecorationSelector.XtermDecoration);
 				e.style.color = '#ffcc00';
+				// must be inlined to override the styles
+				// applied inline in xterm.js
+				e.style.zIndex = '7';
```

Comment:
Is it these that it's overriding?

https://github.com/Tyriar/xterm.js/blob/23328a394bbb10790014d78f888d96174e1aae98/css/xterm.css#L176-L192

If so you can do `z-index: 7 !important` in the CSS class and it'll work.